full/Dockerfile

```diff
@@ -8,6 +8,15 @@ RUN yes | unminimize \
         build-essential \
         htop \
         jq \
+        hub=2.7.0~ds1-1 \
+        zsh=5.5.1-1ubuntu3 \
+        tmux=2.8-3 \
+        fzf=0.17.5-2 \
+        mtr=0.92-2 \
+        ripgrep=0.10.0-2 \
+        neovim=0.3.4-1ubuntu0.19.04.1 \
+        fd-find=7.2.0-2 \
+        silversearcher-ag=2.2.0-1 \
```

Comment:
Maybe we should first decide on a subset in the suggested tools that we estimate are _extremely_ useful?

I don't want to install tons of (sometimes duplicate) tools, which then turn out not to get much usage. 😅

Reply:
Right.  I also feel like we need to evaluate the size of each language, most likely dropping off a ruby version.